index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+
+<html Lang="en-GB">
+
+<head>
+	
```

Comment:
need to add `<meta name="viewport" content="width=device-width, initial-scale=1.0">` tag to help some devices not to display the site weirdly

Reply:
thanks Chris, we haven't covered this yet, when you next pop up would you be able to explain what it is doing. Cheers